src/BooleanSlider/widget/BooleanSlider.js

```diff
@@ -117,8 +117,8 @@ define([
             if (this._contextObj !== null) {
                 dojoStyle.set(this.domNode, "display", "block");
 
-                dojoProp.set(this.trueNode, "textContent", this.trueValue);
-                dojoProp.set(this.falseNode, "textContent", this.falseValue);
```

Comment:
I'm not sure why this is replaced with ``"text"``, because dojo-prop clearly uses ``textContent``. See [source](https://github.com/dojo/dojo/blob/master/dom-prop.js#L151)